### ISSUE TITLE
Install for the 2017 release has a script pyi_rth_pkres error

### ISSUE THREAD

**Issue by dickinsonre** [USEPA/SWMM-EPANET_User_Interface#283]
Hi, I have a 3rd party question (it installs fine for my PC).
His comments are:

I get the following error: 
![www](https://user-images.githubusercontent.com/5599939/43079476-3d758544-8e5b-11e8-98e0-67b18bf015d2.png)

I installed this on both Windows 7 and 10. The Windows 10 application opens, whereas the Windows 7 install gives me the above error on open. The Windows 10 installation will not simulate a model.

Where do I get the current version of QGIS?

**Comment by TongZhai** [comment]
@dickinsonre I assume you are referring to the MTP4r2 version of the installer. Is this SWMM or EPANET? I have a hunch it was SWMM not simulate a model? we just fixed a SWMM engine call in the code that might make a difference. By the way, we have successfully tested installation on both Win 7 and 10. On that failed machine, does he have a full QGIS installation? version? we will need a bit more information about the software environment on that machine. The current version of QGIS is [3.2.2 Bonn](https://qgis.org/en/site/).
